atom/common/api/BUILD.gn

```diff
@@ -0,0 +1,11 @@
+import("//mojo/public/tools/bindings/mojom.gni")
+
+mojom("mojo") {
```

Comment:
when we had `atom/common/api/api_messages.h` it was based on the directory structure following upstream ipc codebase. WDYT about defining browser interfaces under `atom/browser` and renderer interfaces under `atom/renderer`, since in mojo world these are two distinct services.

Reply:
Hmm, looking at `//chrome`, the vast majority of mojom files are under `//chrome/common`. `//chrome/renderer` contains _no_ mojom files. Most of the mojom files in `//chrome/browser` are webui-related.

I think these should be in common because the interfaces are _shared_ between the browser and renderer (one implementing, one using).